Packages/Primitives/Sources/TransactionPreload.swift

```diff
@@ -45,12 +45,15 @@ public struct TransactionPreload {
 public struct SignerInputToken {
     public let senderTokenAddress: String
     public let recipientTokenAddress: String?
-    
+    public let tokenProgram: SolanaTokenProgram
+
     public init(
         senderTokenAddress: String = "",
-        recipientTokenAddress: String? = .none
+        recipientTokenAddress: String? = .none,
+        tokenProgram: SolanaTokenProgram = .token
```

Comment:
let's avoid default `.token` option here, it should be always specified

Reply:
This one is used as `block` and native transfer too, maybe should set as optional